docs/src/advanced/1_variants.md

```diff
@@ -0,0 +1,267 @@
+
+# Screening model variants
+
+A major goal of COBREXA is to make exploring of many model variants easy and
+fast.
+
+One main concept that can be utilized for doing that is implemented in function
+[`screen`](@ref), which takes your model, a list of model _variants_ that you
+want to explore by some specified _analysis_, and schedules the analysis of the
+model variants on all available distributed workers.
+
+In the most basic form, using the slightly simplified variant of
+[`screen`](@ref) that is called [`screen_variants`](@ref) may work as follows:
+
+```julia
+m = load_model(StandardModel, "ecoli_core_model.json")
+
+screen_variants(
+    m,    # the model for screening
+    [
+        [],    # a variant with no modifications
+        [with_set_bound("CO2t", lb = 0, ub = 0)],   # disable CO2 transport
+        [with_set_bound("O2t", lb = 0, ub = 0)],   # disable O2 transport
```

Comment:
to be super clear here maybe for the last array have: `[with_set_bound("O2t", lb = 0, ub = 0), with_set_bound("PFL", lb = 0, ub = 0)] # disable O2 transport and disable PFL (pyruvate formate lyase)`

Reply:
So that it is _super_ obvious that many things can be done

Reply:
any specific reason to select PFL?

^ not-a-biologist-question

Reply:
oh no, it is just my go to reaction :)